PR_REVIEW_CHECKLIST.md

```diff
@@ -0,0 +1,98 @@
+
+# Check PR template
+
+1. Make sure the PR template includes screenshots or videos of the changes being tested on **all platforms**. If there are one or more platforms missing, follow up with the contributor to determine why - we shouldn't be merging changes that aren't tested everywhere.
+    - Feel free to ask for help on how to test changes on all platforms.
+2. Make sure the “Fixed Issues” section is formatted correctly. Links should be clickable and lead to the correct issue being fixed by this PR.
+3. Make sure the testing steps are clear and correctly test the changes made. Make note of what’s required for the test account (i.e. do you need to test using a brand new account, have a workspace, have a bank account, etc).
+    - > **Rule of thumb**: Over-explain what the tester needs to do. Do not rely on the tester having existing knowledge of the app.
+      > 
+      > ### Example of bad steps:
+      >
+	  >	1. Go to this page
+	  >	2. Click the button
+      >
+      > ### Example of good steps:
+      >
+      > 1. Log in with an account that has a workspace, or follow `[these steps](link-to-instructions-for setting-up-workspace)` to make one.
+      > 2. Click the settings button on the top left (the one with your avatar)
+      > 3. Navigate to “Your Workspace” > Reimburse Receipts > Connect bank account
+      > 4. Verify that the bank account view opens without errors
+      >
+    - Take note of the distinction between testing _locally_ and _on staging_. Note: The staging site references the production version of the API.
+4. Make sure the testing steps also cover possible regressions.
+    1. Cases where we definitely need new regression tests:
+        1. Building a new page
+        2. Building a new feature
+        3. Building a new component?
+    2. Cases where we might need new regression tests:
+        1. Bug was easy to find
+5. Make sure that the test includes both **success** and **fail** scenarios.
+    - Ex: if the issue was to prevent the user from saving a field when it is empty, besides testing that we should also test that they can save the field _if it is not empty_.
+
+# Testing the changes
+
+1. Make sure there are no console errors related to the PR. If you find console errors in a branch you’re reviewing, check if they also exist in `main`.
+    1. If they do, proceed as normal (report them in #expensify-open-source if you think they need to be fixed).
+    2. If the errors do not exist in `main`, report it to the PR author - they need to be fixed before the PR can be merged.
+2. Test the changes on **all platforms**. Follow this guide (needs link) to set up each platform.
+    - If you’re unable to boot a platform for any reason, ask for help in the #expensify-open-source Slack channel. Your issue might already have been addressed there, so be sure to search first.
+3. Should we add regression testing for code introduced in this PR?
```

Comment:
Some discussion about regressions [here](https://docs.google.com/document/d/1G-WFzlsc0wqxgbhs8ejP0Ry-vechEIrlKj1FIdaTf20/edit?disco=AAAAUdQgVGw) - between @marcaaron and @Luke9389

Reply:
Still discussing on the doc. At a high level, I think we want to raise awareness of side effects and try to encourage the active prevention of regressions. How exactly that happens has yet to be decided it seems

Reply:
My feelings are the same. If we can't be clear about what we are asking people to do - I'm not sure we should ask them to do it at all. That introduces ambiguity and seems like the opposite of what we are trying to achieve with these guidelines.

Reply:
Killing all regression test mentions for now - will bring up this convo again later in Slack :)